recipes/django-readers/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "django-readers" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/django-readers-{{ version }}.tar.gz
+  sha256: 83ca6d4c9011b640794e2c54c5d8e17bc570c91f43796edad00d6505c8fb4aee
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - django >=2.2
+    - python >=3.6
+
+test:
+  imports:
+    - django_readers
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.django-readers.org
+  summary: A lightweight function-oriented toolkit for better organisation of business logic and efficient selection and projection of data in Django projects.
```

Comment:
This summary is quite big! Almost a description, maybe we can shorten it a bit?

Reply:
Yup; done.